apps/marketplace/components/PrivateRoute.js

```diff
@@ -10,7 +10,7 @@ const PrivateRouteComponent = props => {
       {...rest}
       render={values =>
         loggedIn
-          ? <Component {...values} />
+          ? <Component {...values} {...props} />
```

Comment:
looks like the component is receiving itself as props 🤔. `...rest` should be enough